_pages/en/it-picture-long-term.md

```diff
@@ -26,24 +27,24 @@ In the following sections, we're trying to take the same end state and create "v
 
 ### Citizens
 
-- Cyber security and privacy - citizens have a choice to share their personal data for convenience
+- Cyber security and privacy: citizens have a choice to share their personal data for convenience
 - Generational change: Citizens are used to deal with technologies, they do not face technical illiteracy like 2019.
+- One GC: Citizens do not interact with individual departments, they interact with federal government's services without having to understand the way those are administered.
```

Comment:
Which looks like ... what? In 10 years? Through a government portal, or integrated with existing services? 

To quote from our meeting, "Tangible tangible."

Reply:
I thought that is tangible. It's the OneGC.
Citizens do not see ESDC. They see Federal Government.

Reply:
will reword to clarify